File: js/058_LengthofLastWord.js
/**
 * @param {string} s
 * @return {number}
 */
var lengthOfLastWord = function(s) {
    var len = s.length - 1;
    var count = 0;
    while(s[len] === ' ') --len;
    while(len >= 0 && s[len--] !== ' '){
        ++count;
    }
    return count;
};

File: cpp/058_LengthofLastWord.cpp
class Solution {
public:
    int lengthOfLastWord(string s) {
        int len = s.length() - 1;
        int count = 0;
        while(s[len] == ' '){
            len--;
        }
        while(len >= 0 && s[len] != ' '){
            count++;
            len--;
        }
        return count;
    }
};